WordPress/gutenberg
# KeyboardShortcuts: Update delete shortcut to use `shift + Backspace`

Fixes: #68139 

## What, Why & How?
Previously, the shortcut to remove the `selected blocks` was `Control` `Option` `Z` on Mac, or `Control` `Alt` `Z` on Windows.

This PR introduces the usage of `shift` + `Backspace` for performing the same.

## Testing Instructions
1. Create a block.
2. Press the key combination `SHIFT` + `Backspace` and observe the removal of the block.

## Screencast


https://github.com/user-attachments/assets/0600431d-8b16-44f1-abf0-ed58a455617a





## Screenshot

![Screenshot 2024-12-24 at 3 15 55 PM](https://github.com/user-attachments/assets/da293c9a-5ad7-4d9f-ad09-ce5e1cb7fd93)
